MovieApp/MovieApp/App/Features/Movies/Components/DetailsComponents/CastView/ActorsViewModel/ActorsViewModel.swift

```diff
@@ -0,0 +1,5 @@
+struct ActorsViewModel {
+
+    let actors: [ActorViewModel]
+    
+}
```

Comment:
Why don't you have a variable actors that is of type [ActorViewModel] inside detailsViewModel?
This way you have a struct that has a single variable which is of no use

Reply:
If you look at last screenshot in our slack convo, I had that variable.

It seems I misunderstood your advice because that practice made more sense to me also